packages/rafz/src/index.ts

```diff
@@ -201,7 +212,13 @@ function eachSafely<T>(values: Eachable<T>, each: (value: T) => void) {
 /** Tree-shakable state for testing purposes */
 export const __raf = {
   /** The number of pending tasks */
-  count: 0,
+  count(): number {
+    return pendingCount
+  },
+  /** Whether there's a raf update loop running */
+  isStarted(): boolean {
```

Comment:
I'm wondering if `isRunning` is a little clearer here?

Reply:
Fixed